views/partials/head.ejs

```diff
@@ -1,4 +1,4 @@
-<title></title>
+<title>Movie DB</title>
```

Comment:
can you add a variable before movie db like this <title><%= page_title %> | Movie DB </title> and then pass the title into to all the page views